src/webgl.js

```diff
@@ -281,8 +281,8 @@ function init_conv1_1_program(gl) {
     float outX = float(gl_FragCoord[0]);
     float outY = float(gl_FragCoord[1]);
 
-    float inWidth = videoRes.x + 8.0;
-    float inHeight = videoRes.y + 8.0;
+    float inWidthInverse = 1.0 / (videoRes.x + 8.0);
+    float inHeightInverse = 1.0 / (videoRes.y + 8.0);
```

Comment:
How sure are we that doing a division-based inversion is fast? If it's not, then I think you just moved the division rather than removing it. I think this expression could probably be fed as a uniform though, maybe?

Reply:
This is also another of many possible places where a vector op is probably faster: ```
vec2 inInverse = 1.0 / (videoRes + 8.0);

Reply:
Further to the `1/x` question, it's *possible* that `pow(x, -1)` is faster, but no one anywhere will clearly state. I should probably call it a micro optimization and leave it alone for now :)

Reply:
> How sure are we that doing a division-based inversion is fast? If it's not, then I think you just moved the division rather than removing it. I think this expression could probably be fed as a uniform though, maybe?

These vars get used in the loop so this way we amortize the cost of the divide. But passing it in as a uniform is a nice idea, I'm going to look into that.